adding-permissions.md

```diff
@@ -2,39 +2,117 @@
 
 "Permissions" are what a given user is "permitted" to do.
 
-Often permissions are very nuanced: an app can have several "roles", like "admin", "member", and "super-admin", all of which are permitted to do different things.
-
-We're going to keep Tunr more general, and add two "sets" of permissions:
+Often permissions are very nuanced: an app can have several "roles", like "admin," "member" and "super-admin," all of which are permitted to do different things.
 
+We're going to keep Tunr more general, and add two "sets" of permissions...
 - Only signed-in users can CRUD Artists and Songs
 - A User can edit only their own profile -- not another User's
 
-## 1. Hide Artist/Song create/update/delete links for signed-out users
+## 1. Hide Create/Update/Delete Links for Signed-Out Users
+
+If you're signed out, you should not be able to create, update or delete any Artist or Song from the app.
+
+One way to begin to hide this functionality from signed-out users is to never present them with the links that correspond with those actions in the first place.
+
+We can do that by leveraging the `@current_user` variable we defined in the "Adding Sessions" section. Here's an example of how we would use it to hide the "Add Artist (+)" option from the Artist index page.
```

Comment:
not sure if this is in application controller or not yet - but would be a great place for this method